services/frontend-v3/src/components/profileForms/employmentDataForm/EmploymentDataForm.jsx

```diff
@@ -29,64 +29,47 @@ const EmploymentDataForm = ({ formType }) => {
 
   // Get substantive level options
   const getSubstantiveOptions = useCallback(async () => {
-    const result = await axios.get(`${backendAddress}api/option/getTenure`);
-
-    const options = [];
-    // Generate the data for dropdown
-    for (let i = 0; i < result.data.length; i += 1) {
-      const option = {
-        title: result.data[i].description[locale],
-        key: result.data[i].id,
-      };
-      options.push(option);
-    }
-    setSubstantiveOptions(options);
-    return 1;
+    const result = await axios.get(
+      `${backendAddress}api/option/tenures?language=${
+        locale === "en" ? "ENGLISH" : "FRENCH"
```

Comment:
I noticed that we setting the locale to "ENGLISH" and "FRENCH" or are we still using en and fr ?

Reply:
So I double checked and the language is locale seems to be stored as en. I wonder if its better to change either the frontend or the backend so we stay consistant

Reply:
I think it would be easier to change it in the frontend since we are just setting it in one place, and if we were to change the backend to accept fr or en, we would need to change quite a few things (I think) should the change be in this pr?

Reply:
I think a frontend change would be better since most of the backend is built to use the full word. If the frontend change is minor I think we could do it in this pull request so its all done at once.